src/CheckoutSdk/Sessions/SessionsClient.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Checkout.Common;
+using Checkout.Sessions.Channel;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Checkout.Sessions
+{
+    public class SessionsClient : AbstractClient, ISessionsClient
```

Comment:
There are missing Checkout Validation utils in all methods.

Reply:
I added the validations I found in Java Project. Can you name me which one should I add?